frontend/public/components/utils/documentation.tsx

```diff
@@ -7,7 +7,7 @@ import { FLAGS, connectToFlags } from '../../features';
 export const tectonicHelpBase = 'https://coreos.com/tectonic/docs/latest/';
 
 // TODO: This needs to change depending on the version.
-const openshiftHelpBase = 'https://docs.openshift.org/latest/';
+export const openshiftHelpBase = 'https://docs.openshift.org/latest/';
```

Comment:
Would be nice if we did some factoring here to ensure we avoid typos & other mistakes.  IE, if we add a big `openshift` block:

```JavaScript 
const protocol = 'https://';  // all https
const openshiftDocsBase = `${protocol}docs.openshift.org`;
const openshiftHelpBase = `${openshiftDocsBase}/latest`;
// build all these strings with templates, even the coreOS below, and 
// not have to worry about small errors from a missing https, etc.
```

Reply:
I don't know that you have to kick this off here, just thinking out loud.

Reply:
I'm not sure I agree. This seems more error prone to me 

This will eventually need to be passed as a command line arg, so we can't build it up like this either way.

Reply:
Not sure why we can't build it up, but I prob just don't know/understand the use case you are referring to.

Either way, not a big deal, can def pass over this comment.

Reply:
Because the doc url can be changed so it has to come from a config file